### ISSUE TITLE
Pre production environment on AppStore

### ISSUE THREAD

**Issue by jorgej-ramos** [RadarCOVID/radar-covid-ios#37]
I get the feeling that the pre-production version has been uploaded to the AppStore. This capture is from a few minutes ago. I'm wrong?

<img src="https://user-images.githubusercontent.com/1220565/97702419-3c684f00-1aaf-11eb-9fa1-3223ca9ac8d4.jpeg" width="300" />

**Comment by jorgej-ramos** [comment]
Seems to be working now. Hot fixed server side I suppose. This issue can be closed.

<img src="https://user-images.githubusercontent.com/1220565/97703447-f14f3b80-1ab0-11eb-8e25-c56d5c32b578.PNG" width="300" />

**Comment by jorgej-ramos** [comment]
Just to be a little more precise, it seems that the error is fixed when restarting the application. Maybe these two lines in `HomeViewController` have something to do:

```
resetDataButton.isHidden = !(Config.environment == "PRE")
envLabel.isHidden = !(Config.environment == "PRE")
```
Somehow, in the life cycle of the application when updating, the environment is PRE to, when restarting it, it shows the correct PRO environment.

I suggest some kind of race condition or lifecycle interruption in `HomeViewController`.

**Comment by pvieito** [comment]
Yes, same here. The first time I opened the app after the update the UI was in PRE mode.

**Comment by j-rivero** [comment]
Is the procedure of releasing/uploading open/documented? We might be able to patch/change the procedure to avoid these kind of problems.

**Comment by pantic79** [comment]
Fixed on 07e5fd4321c44e4c2925661fba532208098c4a5a and released 1.1.1 APP verison.

Thanks.

**Comment by pantic79** [comment]
Closing the issue once we have checked that the new version 1.1.1 from App store is working as expected.

Thanks.
